core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -336,186 +337,164 @@
 
   public static ConfigDef baseSchemaRegistryConfigDef() {
 
-    return baseConfigDef()
-        .defineOverride(PORT_CONFIG, ConfigDef.Type.INT, SCHEMAREGISTRY_PORT_DEFAULT,
-            ConfigDef.Importance.LOW, PORT_CONFIG_DOC
-        )
-        .defineOverride(LISTENERS_CONFIG, ConfigDef.Type.LIST, SCHEMAREGISTRY_LISTENERS_DEFAULT,
-            ConfigDef.Importance.HIGH, LISTENERS_DOC + "\n\n"
-                                       + "Schema Registry identities are stored in "
-                                       + "ZooKeeper and are made up of a hostname and "
-                                       + "port. "
-                                       + "If multiple listeners are configured, the "
-                                       + "first listener's port is used for its "
-                                       + "identity."
-        )
-        .defineOverride(RESPONSE_MEDIATYPE_PREFERRED_CONFIG, ConfigDef.Type.LIST,
-            PREFERRED_RESPONSE_TYPES,
-            ConfigDef.Importance.HIGH,
-            RESPONSE_MEDIATYPE_PREFERRED_CONFIG_DOC
-        )
-        .defineOverride(RESPONSE_MEDIATYPE_DEFAULT_CONFIG, ConfigDef.Type.STRING,
-            SCHEMA_REGISTRY_MOST_SPECIFIC_DEFAULT,
-            ConfigDef.Importance.HIGH,
-            RESPONSE_MEDIATYPE_DEFAULT_CONFIG_DOC
-        )
-        .define(KAFKASTORE_CONNECTION_URL_CONFIG, ConfigDef.Type.STRING, "",
-            ConfigDef.Importance.HIGH, KAFKASTORE_CONNECTION_URL_DOC
-        )
-        .define(KAFKASTORE_BOOTSTRAP_SERVERS_CONFIG, ConfigDef.Type.LIST, "",
-            ConfigDef.Importance.MEDIUM,
-            KAFKASTORE_BOOTSTRAP_SERVERS_DOC
-        )
-        .define(SCHEMAREGISTRY_ZK_NAMESPACE, ConfigDef.Type.STRING,
-                DEFAULT_SCHEMAREGISTRY_ZK_NAMESPACE,
-                ConfigDef.Importance.LOW, SCHEMAREGISTRY_ZK_NAMESPACE_DOC
-        )
-        .define(SCHEMAREGISTRY_GROUP_ID_CONFIG, ConfigDef.Type.STRING, "schema-registry",
-                ConfigDef.Importance.MEDIUM, SCHEMAREGISTRY_GROUP_ID_DOC
-        )
-        .define(KAFKASTORE_ZK_SESSION_TIMEOUT_MS_CONFIG, ConfigDef.Type.INT, 30000, atLeast(0),
+    return baseConfigDef(
+        SCHEMAREGISTRY_PORT_DEFAULT,
+        SCHEMAREGISTRY_LISTENERS_DEFAULT,
```

Comment:
We're going to lose the SR-specific documentation for the `listeners` config. Is that what we want, or are we sacrificing that in order to keep the `baseConfigDef(...)` methods simple?

Reply:
i mean... i'm fine with it 👼 

if you aren't i can try to figure out someway to pass this info up the chain....

or maybe we can just always print out an `INFO` with this info at startup?

Reply:
@mageshn, any preferences? I guess I'm fine with losing the SR-specific documentation for the `listener` config, but do you think it's worth adding an INFO level log message, especially considering it's only useful when SR is actually using ZK.

Reply:
ya'll ok with me getting this merged and we figure out logging later?

theres some downstream consumers of this pr that are anxious

Reply:
The SR-specific documentation is kind of incomplete today. It also addresses how the listener is used when someone is using ZK based election. I'm fine with losing this SR specific doc and I don't see a value in the log message.